problem1.py

```diff
@@ -0,0 +1,65 @@
+class MyHashMap:
+    
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        self.array = [-1 for i in range(1000000)]
+        
```

Comment:
Nice try, Mangesh! But try the approach taught in class as well. Helps you better understand